src/util/sqlhelper.cpp

```diff
@@ -182,6 +182,9 @@ namespace hsql {
 
   void printSelectStatementInfo(const SelectStatement* stmt, uintmax_t numIndent) {
     inprint("SelectStatement", numIndent);
+    if (stmt->selectDistinct) {
+      inprint("distinct", numIndent + 1);
```

Comment:
I'm not sure about the scope of this PR. If the scope is not only printing the `distinct` information but also improving the `printSelectStatementInfo`, printing `SelectStatement::withDescriptions` is also still missing in `printSelectStatementInfo`.

Reply:
there is also `hsql::Expr::distinct` for cases such as `SELECT COUNT(DISTINCT column) ...`. If `DISTINCT` is going to be output, shouldn't this case be too?